### ISSUE TITLE
bug: Cannot bind ReactiveForms to ion-input

### ISSUE THREAD

**Issue by iconio** [ionic-team/ionic#21473]
# Bug Report

**Ionic version:**

[ ] **4.x**
[x] **5.x**
**Current behavior:**
There is an error when trying to bind `formControlName` to a `ion-input` on the latest version of Ionic.
The error is:
```
ERROR Error: No value accessor for form control with name: 'email'
ERROR Error: No value accessor for form control with name: 'password'
```

**Expected behavior:**
To bind the data.

**Steps to reproduce:**
Just create a ReactiveForm.

**Related code:**
On the component.ts
```
ngOnInit() {
    this.registerForm = new FormGroup({
        email: new FormControl(),
        password: new FormControl()
    });
}
```

On the HTML
```
<form [formGroup]="registerForm" (ngSubmit)="onRegisterUser()">
    <ion-item lines="full" color="secondary">
        <ion-icon name="mail-outline"></ion-icon>
        <ion-input placeholder="Email" type="email" formControlName="email"></ion-input>
    </ion-item>
    <ion-item lines="full" color="secondary">
        <ion-icon name="lock-closed-outline"></ion-icon>
        <ion-input placeholder="Password" type="password" formControlName="password"></ion-input>
    </ion-item>
    <ion-button expand="full" type="submit" color="primary">REGISTER</ion-button>
</form>
```

**Other information:**
If I replace `ion-input` for `<input>`, the regular HTML tag, it works fine.
I do have `ReactiveFormsModule` imported and declared on my app.module.

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 6.10.0 (/Users/wd/.nvm/versions/node/v12.11.1/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.1.1
   @angular-devkit/build-angular : 0.901.7
   @angular-devkit/schematics    : 9.1.7
   @angular/cli                  : 9.1.7
   @ionic/angular-toolkit        : 2.2.0

Capacitor:

   Capacitor CLI   : 2.1.2
   @capacitor/core : 2.1.2

Utility:

   cordova-res (update available: 0.14.0) : 0.8.1
   native-run (update available: 1.0.0)   : 0.3.0

System:

   NodeJS : v12.11.1 (/Users/wd/.nvm/versions/node/v12.11.1/bin/node)
   npm    : 6.14.5
   OS     : macOS Catalina
```

**Comment by NikolaPeevski** [comment]
I copied and paste your code @iconio works without any  issues. Here's a clean slate repo: https://github.com/NikolaPeevski/ion-reactive-forms

```

Ionic:

   Ionic CLI                     : 6.10.1 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.2.1
   @angular-devkit/build-angular : 0.901.8
   @angular-devkit/schematics    : 9.1.8
   @angular/cli                  : 9.1.8
   @ionic/angular-toolkit        : 2.2.0

Utility:

   cordova-res (update available: 0.15.0) : 0.9.0
   native-run (update available: 1.0.0)   : 0.3.0

System:

   NodeJS : v10.21.0 (/usr/local/Cellar/node@10/10.21.0/bin/node)
   npm    : 6.14.4
   OS     : macOS Catalina
```

**Comment by liamdebeasi** [comment]
Thanks for the issue! This issue is being closed due to the lack of a code reproduction. If this is still an issue with the latest version of Ionic, please create a new issue and ensure the template is fully filled out.

Thank you for using Ionic!
